cobbler/module_loader.py

```diff
@@ -63,7 +63,7 @@ def load_modules(self):
             basename = filename.replace(self.mod_path, "")
             modname = ""
 
-            if basename in ("__pycache__", "__init__.py"):
+            if "__pycache__" in basename or "__init__.py" in basename:
```

Comment:
Do we expect that `basename` contains `"__pycache__"` but is not `"__pycache__"`? If not, I'd use `==` to communicate that `basename` is not a substring.

Same for `__init__.py`. Here it could be different (`__init__.pyc` maybe?) I guess.

Reply:
Yes I expect it to contain these as we strip the module directory from the path. Since plugins might be recursive we get somthing alike `plugina/__pycache__` for example. Same is then also valid for `__init__.py`.